bin/oneoff/move_storage_apps_to_dashboard.rb

```diff
@@ -0,0 +1,69 @@
+require File.expand_path('../../../pegasus/src/env', __FILE__)
```

Comment:
Not critical, but we have a convention to make our one-off scripts shell scripts by setting the first line to:

`#!/usr/bin/env ruby`

And making the file executable (an attribute that can be committed in git):
`chmod +x bin/oneoff/move_storage_apps_to_dashboard.rb`

And typically renaming to drop the `.rb` extension.

Reply:
@sureshc when I run `chmod +x bin/oneoff/move_storage_apps_to_dashboard.rb` I don't see any changes to commit, should I?

Reply:
Try setting `filemode = true` in `[repository]/.git/config`?

Reply:
Look like this command would accomplish the same:
`git config core.filemode true`